### ISSUE TITLE
Not sure how to handle/fix "template must end with newline"

### ISSUE THREAD

**Issue by boris-petrov** [ember-template-lint/eslint-plugin-hbs#33]
I use inline templates only in tests and there I have something like:

```javascript
await render(hbs`{{some-component this.someValue`);
```

And ESLint complains:

```
13:18  error  1 error(s): template must end with newline  hbs/check-hbs-template-literals
```

Putting an `\n` at the end doesn't seem to silence it. Besides, I don't want to do that in all places. Apart from having a different config for ESLint's template linting, can something be done in the plugin?

**Comment by rwjblue** [comment]
Ya, we should probably disable that rule for inline templates.

**Comment by jaydgruber** [comment]
So if #35 goes in and the template-linter has access to the filePath for inline templates, we could either:
- update the blueprint to set `insert_final_newline = false` for some pattern `**/tests/**/*.js` or something (since `editorconfig` is the default config for the rule now)
- update the `eol-last` rule directly to use the filePath and ignore any `.js`
  - I'm not sure what other use cases besides tests folks have for inline templates. Should this still be configurable?

**Comment by rwjblue** [comment]
I think the rule should be aware that the file path is not `.hbs` and allow relaxing the rule. You could probably do that with overrides already.

**Comment by jaydgruber** [comment]
yeah, overrides would work. although #35 / (fixing #40) still needed, b/c it is hard to do overrides without the filePath right now
